nilearn/tests/test_signal.py

```diff
@@ -358,12 +379,13 @@ def test_clean_confounds():
 
     # Test without standardizing that constant parts of confounds are
     # accounted for
-    np.testing.assert_almost_equal(nisignal.clean(np.ones((20, 2)),
-                                                  standardize=False,
-                                                  confounds=np.ones(20),
-                                                  detrend=False,
-                                                  ).mean(),
-                                   np.zeros((20, 2)))
+    for normalize in [None, 'psc']:
```

Comment:
I included PSC in the previous test for constant confound set in PR #963 and the test is passing.
The Travis failing test is test_testing.test_memory_usage and I guess it is related to issue #986 and not specific to this PR.